go-app-ussd_pmtct_rapidpro.js

```diff
@@ -169,14 +169,14 @@ go.app = function() {
         self.add("state_enter_msisdn", function(name) {
             return new FreeText(name, {
                 question:
-                    "Please enter the cell number of the mother who would like to sign up to " +
-                    "receive HIV-related messages from MomConnect e.g. 0813547654.",
+                    "Please enter the cell number of the mom that would like to sign up to or " +
+                    "opt out of HIV-related messages from MomConnect e.g. 0813547654.",
                 check: function(content) {
                     if(!utils.is_valid_msisdn(content, "ZA")) {
                         return (
-                            "Sorry, we don't understand that cell number. Please enter 10 digit " +
-                            "cell number that the mother would like to get HIV-related messages " +
-                            "on, e.g. 0813547654.");
+                            "Sorry, we don't understand that number. Please enter 10 digit " +
```

Comment:
hmm over here we've got number, instead of no. or cell number.

Reply:
with this, there was a character limit issue, But we further state that we need the cell number in the next sentence `Please enter 10 digit cell number that the mother would like to get or opt out of HIV-related messages ` it clarifies to the user what is needed as input